src/System.Runtime.WindowsRuntime/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      uap-Windows_NT;
+      uapaot-Windows_NT;
```

Comment:
I don't know. Was just copying what the source project had. Does it matter for tests?